Language/Functions/Zero, Due, MKR Family/analogReadResolution.adoc

```diff
@@ -31,7 +32,7 @@ The *Due, Zero and MKR Family* boards have 12-bit ADC capabilities that can be a
 
 [float]
 === Parameters
-`bits`: determines the resolution (in bits) of the value returned by the `analogRead()` function. You can set this between 1 and 32. You can set resolutions higher than 12 but values returned by `analogRead()` will suffer approximation. See the note below for details.
+`bits`: determines the resolution (in bits) of the value returned by the `analogRead()` function. You can set this between 1 and 32. You can set resolutions higher than the supported 10,12 or 16 bits, but values returned by `analogRead()` will suffer approximation. See the note below for details.
```

Comment:
```suggestion
`bits`: determines the resolution (in bits) of the value returned by the `analogRead()` function. You can set this between 1 and 32. You can set resolutions higher than the supported 12 or 16 bits, but values returned by `analogRead()` will suffer approximation. See the note below for details.
```
As far as I can tell, all of the official boards with `analogReadResolution()` have either a 12 or 16 bit ADC. The core libraries of the 10 bit ADC AVR and megaAVR  boards don't have  `analogReadResolution()`.